po/pt_PT.po

```diff
@@ -1,5 +1,4 @@
 # Portugal's Portuguese translations for Git package.
-osgid "set compression level"
```

Comment:
Why introduce this line in heading comments?

Reply:
Make sure to have a valid "po/pt_PT.po" file in each commit.

Reply:
I believe it is `git-po-helper` who is doing this automatically. It is the second time now.
I will fix this

Reply:
it was already corrected in the last commit.
What should I do?

Reply:
I did a `rebase -i` and corrected/amended that commit.
Please fetch my new changes